src/models/IdentityProvider.js

```diff
@@ -105,13 +105,6 @@ class IdentityProvider extends Resource {
     return this.client.getIdentityProviderSigningKey(this.id, keyId);
   }
 
-  /**
-   * @param {string} keyId
-   */
-  deleteSigningKey(keyId) {
```

Comment:
This alias method was added by myself, then I realized it's only delete operation for global key instead of signing key. 
It's already been delete in `openapi-update` branch, not sure why the diff still show up here.